### PR TITLE
Registry in database with full functionality

### PR DESCRIPTION
**Pull request by daniel-thom**
Restores full functionality with the registry database.

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -139,171 +144,31 @@ this command after modifiying the paths specified in the JSON5 file. Four `bigme
...
-`dsgrid-test-data/filtered_registries/simple_standard_scenarios/dump`, push to GitHub and open a
-pull request.
+3. Copy (e.g., cp, scp, rsync) or arangodump the output files (registry JSON and Parquet) from the
+previous step to a dsgrid-test-data repository in the directory
+`dsgrid-test-data/filtered_registries/simple_standard_scenarios`. Edit the file
+`dump/data_path_*.data.json` to ensure that the `data_path` value is set to
+`dsgrid-test-data/filtered_registries/simple_standard_scenarios`. Make a branch, commit, push
+it to GitHub, and open a pull request.
 
 ### Run tests
```
Comment (elainethale) [inline dev/README.md:173]: To run tests on Eagle, I assume one needs to get a compute node. Is it also required/advisable to set up a spark cluster first? Is one node for one hour sufficient? Is any ArangoDB set up required? Is it possible to browse the test registry after the tests have run? If so, how?

File: dev/README.md [hunk 2]
```diff
@@ -124,7 +124,12 @@ contains a JSON-exported database that you must import into your local ArangoDB
 
 You can use a native ArangoDB installation or the docker container.
 ```
-$ arangorestore --create-database --input-directory dsgrid-test-data/filtered_registries/simple_standard_scenarios/dump --server.database simple-standard-scenarios --include-system-collections true
+$ arangorestore \
+    --create-database \
+    --input-directory \
+    dsgrid-test-data/filtered_registries/simple_standard_scenarios/dump \
+    --server.database simple-standard-scenarios \
+    --include-system-collections true
```
[the remark this answers is not in the document]
Comment (daniel-thom) [reply dev/README.md:132]: The command is correct. The error means that the arango code does not have access to the dsgrid-test-data directory. Did you bind-mount that directory when you started the container? Maybe the path needs adjustment?

Yes, when you run `pytest` (or any dsgrid action when working on a local system) you have to have an instance of ArangoDB running. An alternate implementation would be for us to fully manage Arango within the test framework. I'm not liking that option currently. I'll add some clarifications to the README.

File: dsgrid/api/app.py [hunk 3]
```diff
@@ -67,15 +66,11 @@
 # subprocesses that run queries.
 # If both processes try to use the Hive metastore, a crash will occur.
 spark = init_spark("dsgrid_api", check_env=False)
+conn = DatabaseConnection.from_url(DSGRID_REGISTRY_DATABASE, database="simple-standard-scenarios")
```
[the remark this answers is not in the document]
Comment (daniel-thom) [reply dsgrid/api/app.py:69]: bug

File: dsgrid/tests/restore_simple_standard_scenarios_db.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+import json
+import re
+from pathlib import Path
+
+from dsgrid.registry.common import Edge
+from dsgrid.registry.registry_database import RegistryDatabase, DatabaseConnection
+
+
+def restore_simple_standard_scenarios(conn=None):
```
[the remark this answers is not in the document]
Comment (daniel-thom) [reply dsgrid/tests/restore_simple_standard_scenarios_db.py:11]: I attempted to give the reason for this in the module docstring. And then the GitHub Action file has
```
        # Haven't figured out how to run arangorestore here. Use this instead.
        python dsgrid/tests/restore_simple_standard_scenarios_db.py
```
I tried and failed to get arangorestore in the docker container to authenticate with the database and find the dsgrid-test-data folder. It took a lot less time to import that data into the database this way.

File: scripts/create_simple_standard_scenarios.sh [hunk 5]
```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+export SPARK_CLUSTER=spark://$(hostname):7077
+export SPARK_CONF_DIR=$(pwd)/conf
+export REPO_BASE=${HOME}/repos
```
[the remark this answers is not in the document]
Comment (elainethale) [reply scripts/create_simple_standard_scenarios.sh:4]: Yes, but the ones I marked aren't always immediately clear. I think they'll be okay if DSGRID_REPO points to a folder named dsgrid.

**Review by elainethale — APPROVED** [review]
Tests run for me on Eagle now, and the situation on my laptop is better than it was before.

**Comment by elainethale** [comment]
@daniel-thom, I'm good with this now that I can run tests on Eagle and my laptop is behaving better. Do you think you'll merge it in today?

**Pull request merged**